Ports/ObjFW/package.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port='ObjFW'
-version='8d19ba9c8f1955673569e10919025624975e896f'
+version='1.1.2'
 useconfigure='true'
 files=(
-    "https://github.com/ObjFW/ObjFW/archive/${version}.tar.gz#4fbdeba8f2792f5fcdb75d3f0da12f927ea3b0ec0c0f16bef9a743b476f84724"
+    "https://objfw.nil.im/downloads/objfw-${version}.tar.gz#5d9f9a70d583298e780ae11fc75a7ae2beeef904b301e1bc4f4ffa8d7ee31d9f"
```

Comment:
Since we are using an actual release now, we can remove the autogen usage below.